answerking/settings/base.py

```diff
@@ -154,3 +154,5 @@
 SECURE_FRAME_DENY = False
 SECURE_CONTENT_TYPE_NOSNIFF = False
 SECURE_BROWSER_XSS_FILTER = False
+
+TEST_RUNNER = 'snapshottest.django.TestRunner'
```

Comment:
This may need putting in `settings/development.py`

Reply:
This has been moved :)